easybuild/easyblocks/generic/pythonbundle.py

```diff
@@ -132,7 +132,8 @@ def make_module_extra(self, *args, **kwargs):
         # these .pth files to work as expected. See: https://docs.python.org/3/library/site.html#module-site
         # .pth files always should be in the site folder, so most of the path is fixed.
         # Try the installation directory first
-        if self.installdir and search_file([self.installdir], r".*\.pth$", silent=True):
```

Comment:
Yes, a tuple that contains an empty list, isn't empty and therefore returns True. If you just want to check if `hits` is an empty list, this will do it.